app/code/Magento/Downloadable/view/frontend/templates/email/order/items/creditmemo/downloadable.phtml

```diff
@@ -4,32 +4,37 @@
  * See COPYING.txt for license details.
  */
 ?>
-<?php /** @var $block \Magento\Downloadable\Block\Sales\Order\Email\Items\Downloadable */ ?>
+<?php
```

Comment:
Please remove not needed closing and opening PHP tags

Reply:
fixed